src/components/governance/PageProposal.vue

```diff
@@ -299,12 +305,20 @@ export default {
         `
       },
       variables() {
+        /* istanbul ignore next */
```

Comment:
what's with the Istanbul 🇹🇷comments everywhere?

Reply:
removes codecoverage checks for the next line

Reply:
Yep, I would like to find out how to test that graphql functions ...